website/src/blogs/how-to-install-openebs-on-ibm-cloud-private.md

```diff
@@ -0,0 +1,134 @@
+---
+title: How to install OpenEBS on IBM Cloud Private
+author: Murat Karslioglu
+author_info: VP @OpenEBS & @MayaData_Inc. Lives to innovate! Opinions my own!
+date: 19-11-2017
+tags: Helm Charts, Openebs, Solutions, Icp
+excerpt: What is IBM Cloud Private? IBM Cloud Private (ICP) is a new application platform that is based on Kubernetes and provides services for developing and managing on-premises containerized applications. 
+---
+
+## What is IBM Cloud Private?
+
+**IBM Cloud Private (ICP)** is a new application platform that is based on **Kubernetes** and provides services for developing and managing **on-premises** containerized applications. ICP Community Edition (ICP-CE) is distributed free of charge for non-production use and is available on Docker Hub. For commercial use, you would need the Enterprise package.
+
+In my previous blog post, [Introduction to IBM Cloud Private](http://containerized.me/introduction-to-ibm-cloud-private/), I have covered step-by-step installation of ICP 2.1. This time I will focus on configuring **OpenEBS** as a **persistent storage** option and deploying a stateful workload (MongoDB) using OpenEBS storage classes.
+
+## Prerequisites
+
+### Hardware
+
+- Minimum three x64 servers
+
+### Software
+
+- [Ubuntu Server 16.04 LTS](https://www.ubuntu.com/download/server)
+- IBM Cloud Private 2.1
+- [OpenEBS](https://github.com/openebs/openebs)
+
+### Install IBM Cloud Private
+
+Follow instructions from [Introduction to IBM Cloud Private](http://containerized.me/introduction-to-ibm-cloud-private/) to deploy a multi-node ICP cluster.
+
+### Install OpenEBS on ICP
+
+1. Log in to the ICP console and go to the **Admin/Repositories** menu.
```

Comment:
It's better to use unordered list here rather than ordered list

Reply:
Let's not change the structure of the content as provided by the author.